src/main/java/domain/Validator.java

```diff
@@ -0,0 +1,106 @@
+package domain;
+
+import io.Printer;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class Validator implements ValidatorInterface {
+    private static final int MAX_NAME_SIZE = 5;
+    private static final String valiNumber = "^[0-9]+$";
+    private static final String commaInARow = "^(,,)+$";
+    private static final String characterOTN = "^[a-zA-Z,]+$";
+    Printer printer;
+
+    public Validator() {
+        printer = new Printer();
+    }
+
+    @Override
+    public boolean validateName(String s) {
+        return inputNothing(s)
+                && inputCommaInARow(s)
+                && inputCharactersOtherThanName(s)
+                && startWithComma(s)
+                && endWithComma(s)
+                && this.overSizeCharacters(s)
+                && inputSameName(s);
+    }
+
+    @Override
+    public boolean validateNumber(String s) {
```

Comment:
boolean을 반환하는 함수의 네이밍은 일반적으로 is, can, should, has 등을 사용합니다.
[[참고]](https://jacking75.github.io/ETC_true_false_return_function_naming/)

```suggestion
    public boolean isValidNumber(String s) {
```

만으로도 반환 타입이 무엇인지 나타낼 수 있으니 더 좋지 않을까요?